### ISSUE TITLE
Open Source Mycetoma (MycetOS) Meeting Jan 26/27 2021

### ISSUE THREAD

**Issue by mattodd** [OpenSourceMycetoma/Series-1-Fenarimols#45]
Further to confirmation of meeting times ([here](https://github.com/OpenSourceMycetoma/Series-1-Fenarimols/issues/28#issuecomment-759784857)) and previous meetings (#28) and previous meeting (#39) **_recording?_**, we're all meeting up Jan 26/27 at Zoom ID: 992-8154-9497 https://uni-sydney.zoom.us/j/99281549497.

**Agenda Items:**

Science updates from different sites.
Current needs
Potential applications for research funds.
Newsletter and World NTD day

Anything else? @OpenSourceMycetoma/CoreContrib

**Comment by Wilson-Lm** [comment]
As promised, the latest in vivo figures for both fenarimol analogues and pandemic box. Data can be found in our main excel sheet in GitHub.
[fenarimol 2021 meeting 1.pptx](https://github.com/OpenSourceMycetoma/Series-1-Fenarimols/files/5876524/fenarimol.2021.meeting.1.pptx)

**Comment by MFernflower** [comment]
@Wilson-Lm @wwjvdsande Just wanted to confirm the name of the drug resistant fungus that you found tafenoquine was able to kill

It was the genus Medicopsis in the genus Pleosporales correct?

**Comment by Wilson-Lm** [comment]
> @Wilson-Lm @wwjvdsande Just wanted to confirm the name of the drug resistant fungus that you found tafenoquine was able to kill
> 
> It was the genus Medicopsis in the order Pleosporales correct?

@MFernflower That is correct. Medicopsis romeroi of the Pleosporales! and also Falciformispora senegalensis of the same order.

**Comment by Wilson-Lm** [comment]
_in vitro_ and _in vivo_ potency results + misc information have been included and updated in the main excel file, sheet _In vitro & vivo potency & LogD values_ 
That includes compounds that are active _in vitro_ and also compounds that underwent _in vivo_ screening.

**Comment by MA-Jjingyi** [comment]
Hi,Wlison. good morning.
it's my main topic of my next research, right? oh ,my goodness. I must learn more technics about them in the near future,  i think! thanks again.
________________________________
